bemani/backend/popn/unilab.py

```diff
@@ -321,7 +322,16 @@ def get_common_config(self) -> Tuple[Dict[int, int], bool]:
                 # 1 - Net taisen
                 # 2 - Net taisen + Local mode
                 3: 1 if enable_net_taisen else 0,
-                # Unknown event (0-7)
+                # Paseli Festival Attract mode ads (0-7)
+                # As updates to datecodes happen the older ones are are completely locked out
```

Comment:
It might be cute in the future to check datecode to display the correct values here but eh